addon/utils/prop-types.js

```diff
@@ -134,5 +145,11 @@ PropTypes.oneOf = function (typeDefs) {
   return type
 }
 
+PropTypes.instanceOf = function (typeDef) {
```

Comment:
This is overriding what is set by https://github.com/philpee2/ember-prop-types/blob/7b647c5ad5afe1267779d0992c5086af7152d5af/addon/utils/prop-types.js#L111. If you want to manually set this then you should not put `instanceOf` in the validators object at the top of the function and should instead do something similar to https://github.com/philpee2/ember-prop-types/blob/7b647c5ad5afe1267779d0992c5086af7152d5af/addon/utils/prop-types.js#L114.